OTAcceleratorCore/UI/OTOneToOneCommunicationController.m

```diff
@@ -60,7 +60,7 @@ - (IBAction)publisherCallButtonPressed:(UIButton *)sender {
         }];
     }
     else {
-        [SVProgressHUD popActivity];
+        [SVProgressHUD dismiss];
```

Comment:
@marinaserranomontes  actually, `dismiss` and `popActivity ` are slightly different. `dismiss` will remove the HUD but `popActivity` will decrease the count (HUD goes away only if the count is 0).

Reply:
@Lucashuang0802 I think for this basic sample we had: show and popActivity, so the "dismiss" should be enough, right?
Also, I think it could be a good moment to update and to fix the SVProgress to the last version. "popActivity" method doesn't seem to be in the last version.

Reply:
As a side note, AFAIK, popActivity was removed from SVProgressHUD because it was having problems

Reply:
@robjperez @marcioaffonso I used `popActivity` just for not dismissing the HUD improperly. I did not take another look into that. I am okay if you guys are certain. 😃